src/content/services/monetizationTagManager.ts

```diff
@@ -294,7 +332,14 @@ export class MonetizationTagManager extends EventEmitter {
 
     this.monetizationTags.set(tag, details)
 
-    if (walletAddress) startMonetization({ requestId, walletAddress })
+    if (walletAddress) {
+      startMonetization({ requestId, walletAddress })
```

Comment:
should this be 
```suggestion
      await startMonetization({ requestId, walletAddress })
```

Reply:
we don't want to await this one, just lunch it and solve it whenever